packages/components/addon/components/hds/app-footer/status-link.hbs

```diff
@@ -0,0 +1,23 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Item ...attributes>
```

Comment:
As you already mentioned in your comment, there is an opportunity here to use directly the `<AppFooter::Link>` and override the icon's color via:

- class name for the standard status
- CSS custom prop/variable passed via JS (`style.setProperty(...)`) and used in the CSS to override the `color` property

(unless you want to pass a CSS variable in both cases, which may be simpler, but in that case add a note/comment in the CSS so we know where to find the color definitions for the different statuses)

Reply:
Refactoring may help with implementing the theming so I can look at it as part of that work.